MAX78000_Feather/README.md

```diff
@@ -146,18 +146,15 @@ The SWD interface is unavailable for a certain number of clock cycles after rese
 1.  Remove the USB cable connected to the MAX78000FTHR board.  
 2.  Place the on-board debug adapter in MAINTENENCE mode by holding down button SW5 while reconnecting the USB cable to the host PC.  
    - The debug adapter will enumerate as a mass storage device named MAINTENANCE.  
-   - Drag-n-Drop the provided bin file to the drive named MAINTENANCE:  [special DAPLINK bin file](MAX32625PICO_files/max32625_max32630fthr_if_mass-erase-v1.bin).  
+   - Drag-n-Drop the provided bin file to the drive named MAINTENANCE:  [DAPLINK binary file](https://github.com/MaximIntegratedMicros/max32625pico-firmware-images/blob/main/bin/max32625_max78000fthr_if_crc_v1.0.2.bin).  
    - Following the Drag-n-Drop, the debug adapter should reboot and reconnect as a drive named DAPLINK.  
-
-3.  Make sure the 'Automation allowed' field is set to 1 in the DETAILS.TXT file on the DAPLINK drive.  If not, follow [these instructions](https://https://github.com/ARMmbed/DAPLink/blob/master/docs/ENABLE_AUTOMATION.md) to enable it.  
-4.  Create an empty file named 'erase128.act' and Drag-n-Drop it onto the DAPLINK drive.
+3.  Make sure the 'Automation allowed' field is set to 1 in the DETAILS.TXT file on the DAPLINK drive. If not:
+    - Create and copy an empty text file named '**auto_on.cfg**' to DAPLINK drive while SW5 button is held.
+    - Release SW5 button when the drive unmounts. When it remounts, confirm "Automation Allowed" is set to 1 in DETAILS.TXT file.
+4.  Create an empty text file named '**erase.act**' and Drag-n-Drop it onto the DAPLINK drive.
 5.  This should mass erase the flash of the target device, allowing the device to be programmed again.
-6.  Restore the original debug adapter firmware by performing the following:  
-   - Re-enter MAINTENANCE mode by disconnecting the USB cable to the debug adapter, pressing down on button SW5, and then reconnecting the USB cable.  
-   - Once the MAINTENANCE drive appears, Drag-n-Drop this file: [original DAPLINK bin file](MAX32625PICO_files/max32625pico_daplink.bin).  
-   - This will again reboot the debug adapter and reconnect as DAPLINK.  
 
-At this point, the target device should be once again programmable and the debug adapter restored to its original firmware.  
+At this point, the target device should be once again programmable and no need to restore original firmware of debug adapter.  
```

Comment:
Second sentence not needed, You can remove "... and no need ..." ,

Reply:
Ok